kani-compiler/src/kani_queries/mod.rs

```diff
@@ -2,65 +2,28 @@
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 //! Define the communication between KaniCompiler and the codegen implementation.
 
-use cbmc::{InternString, InternedString};
-use kani_metadata::AssignsContract;
-use std::{
-    collections::HashMap,
-    path::PathBuf,
-    sync::{Arc, Mutex},
-};
+use std::sync::{Arc, Mutex};
 
 use crate::args::Arguments;
 
 /// This structure should only be used behind a synchronized reference or a snapshot.
+///
+/// TODO: Merge this with arguments
```

Comment:
What does that mean? Do we have a GitHub issue to track this?

Reply:
This struct is now redundant, since it is only wrapping Args.

I prefer avoiding GitHub issues for cleanup work that has no effect to the end user.